pkg/ovsdb/dbserver.go

```diff
@@ -19,14 +19,25 @@ import (
 	"github.com/ibm/ovsdb-etcd/pkg/types/_Server"
 )
 
+type DBServerInterface interface {
+	Lock(ctx context.Context, id string) (bool, error)
+	Unlock(ctx context.Context, id string) error
+	AddSchema(schemaName, schemaFile string) error
+	LoadServerData() error
+	GetData(prefix string, keysOnly bool) (*clientv3.GetResponse, error)
+	GetMarshaled(prefix string, columns []interface{}) (*[]map[string]string, error)
```

Comment:
GetMarshaled, will probably  be removed too

Reply:
same as above